src/components/core/geoChart/GeoChartRenderer.tsx

```diff
@@ -0,0 +1,115 @@
+// (C) 2007-2019 GoodData Corporation
+import * as React from "react";
+import isEqual = require("lodash/isEqual");
+import noop = require("lodash/noop");
+import mapboxgl = require("mapbox-gl");
+import { DEFAULT_PUSHPIN_LOCATION } from "../../../constants/geoChart";
+import { IChartConfig } from "../../../interfaces/Config";
+
+import "../../../../styles/scss/geoChart.scss";
+
+const MAPBOX_ACCESS_TOKEN =
+    "pk.eyJ1IjoiaW1udXR6IiwiYSI6ImNrMHAxY2UxZzBnc2EzZG11YmVhd2dubG0ifQ.bUTN7ceAHq6kVooe3MKgqg";
+const MAPBOX_STYLE = "mapbox://styles/mapbox/streets-v11";
```

Comment:
we should put constants to `/gooddata-react-components/src/constants/geoChart.ts`